app/Http/Controllers/LoginController.php

```diff
@@ -81,30 +76,32 @@ public function loginUsingCredentials(LoginRequest $req, User $user, Auth $auth,
     	$auth->successful = 1;
     	$auth->save();
 
-        // We check if user has all fees paid, if not, we auto-suspend him..
-        if(empty($user->is_superadmin)) {
-            $fee->checkFeesForSuspention($user);
-        }
+      // We check if user has all fees paid, if not, we auto-suspend him..
+      $user->checkStillValid();
 
+      //dd($user->getObject());
     	// We try to also add it to session..
-        $userData = $user->getLoginUserArray($loginKey);
+      $userData = $user->getLoginUserArray($loginKey);
 
-		Session::put('userData', $userData);
+      Session::put('userData', $userData);
+      Session::save();
+      //dd(Session::all());
     	// Mirroring Laravel session data to PHP native session.. For use with angular partials..
     	session_start();
     	$_SESSION['userData'] = Session::get('userData');
-    	session_write_close();    	
+    	session_write_close();
 
     	$toReturn = array(
-			'success'	=>	1,
-			'message'	=>	$loginKey
-		);
+  			'success'	=>	1,
+  			'message'	=>	$loginKey
+  		);
 
+      error_log("User login: " . $user->getName());
```

Comment:
Do not use error_log. Use Log::info, warning, etc instead
Details: https://laravel.com/docs/5.4/errors#logging